geode-core/src/main/java/org/apache/geode/management/internal/cli/modes/CommandModes.java

```diff
@@ -86,7 +87,10 @@ private void parseModes(String str) throws JSONException {
   }
 
   private void logException(Exception e) {
-    Cache cache = CacheFactory.getAnyInstance();
+    Cache cache = ManagementAgent.getCache();
```

Comment:
You could probably revert the above line and just keep the if-block you added because it fetches getCacheForProcessingClientRequests.

Reply:
PS: I think I'm mixing up getCacheForProcessingClientRequests and getCacheForProcessingClientAccess. Do we really need both?

Reply:
Thanks Kirk.  Sorry - that request/access confusion was already in the code when I started this work.  I'll make another checkin that clears that up after I address your other comments and fix some failing unit tests.